srcpkgs/base-files/files/locale.sh

```diff
@@ -1,9 +1,15 @@
 # Sets up locale system settings from /etc/locale.conf.
 #
-if [ -s /etc/locale.conf ]; then
+if [ -s "/etc/locale.conf" ]; then
 	. /etc/locale.conf
 fi
 
+# Sets up locale user settings from $HOME/.config/locale.conf.
+#
+if [ -s "$HOME/.config/locale.conf" ]; then
+	. $HOME/.config/locale.conf
```

Comment:
i'd rather not and let the user put it in their shell initialization.

also this will fail for me since i only have $HOME/etc not $HOME/.config

Reply:
Ok. Should I close this?